File: artist-service/src/main/java/com/gallery/artistservice/domain/Artist.java
package com.gallery.artistservice.domain;

import java.time.LocalDate;

public class Artist {
    private ArtistID id;
    private String name;
    private LocalDate birthDate;
    private String birthplace;
    private String nationality;
    private String photo;

    public Artist() {
        this.id = new ArtistID();
        this.name = "";
        this.birthDate = LocalDate.now();
        this.birthplace = "";
        this.nationality = "";
        this.photo = "";
    }

    public Artist(ArtistID id, String name, LocalDate birthDate, String birthplace, String nationality, String photo) {
        this.id = id;
        this.name = name;
        this.birthDate = birthDate;
        this.birthplace = birthplace;
        this.nationality = nationality;
        this.photo = photo;
    }

    public ArtistID getId() {
        return id;
    }

    public void setId(ArtistID id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public LocalDate getBirthDate() {
        return birthDate;
    }

    public void setBirthDate(LocalDate birthDate) {
        this.birthDate = birthDate;
    }

    public String getBirthplace() {
        return birthplace;
    }

    public void setBirthplace(String birthplace) {
        this.birthplace = birthplace;
    }

    public String getNationality() {
        return nationality;
    }

    public void setNationality(String nationality) {
        this.nationality = nationality;
    }

    public String getPhoto() {
        return photo;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }
}

File: app/ArtistService/Domain/Artist.cs
using System;

namespace ArtistService.Domain
{
    public class Artist
    {
        private ArtistID _id;
        private string _name;
        private DateTime? _birthDate;
        private string _birthplace;
        private string _nationality;
        private string _photo;

        public Artist()
        {
            this._id = new ArtistID();
            this._name = "";
            this._birthDate = null;
            this._birthplace = "";
            this._nationality = "";
            this._photo = "";
        }

        public Artist(ArtistID id, string? name, DateTime? birthDate, string? birthplace, string? nationality, string? photo)
        {
            this._id = id;
            this._name = name ?? "";
            this._birthDate = birthDate;
            this._birthplace = birthplace ?? "";
            this._nationality = nationality ?? "";
            this._photo = photo ?? "";
        }

        public Artist(int artistId, string? name, DateTime? birthDate, string? birthplace, string? nationality, string? photo)
        {
            this._id = new ArtistID(artistId);
            this._name = name ?? "";
            this._birthDate = birthDate;
            this._birthplace = birthplace ?? "";
            this._nationality = nationality ?? "";
            this._photo = photo ?? "";
        }

        public Artist(Artist artist)
        {
            this._id = new ArtistID(artist._id);
            this._name = artist._name;
            this._birthDate = artist._birthDate;
            this._birthplace = artist._birthplace;
            this._nationality = artist._nationality;
            this._photo = artist._photo;
        }

        public ArtistID Id
        {
            get { return this._id; }
            set { this._id = value; }
        }

        public string Name
        {
            get { return this._name; }
            set { this._name = value; }
        }

        public DateTime? BirthDate
        {
            get { return this._birthDate; }
            set { this._birthDate = value; }
        }

        public string Birthplace
        {
            get { return this._birthplace; }
            set { this._birthplace = value; }
        }

        public string Nationality
        {
            get { return this._nationality; }
            set { this._nationality = value; }
        }

        public string Photo
        {
            get { return this._photo; }
            set { this._photo = value; }
        }
    }
}
